public/locales/es/common.json

```diff
@@ -9,7 +9,17 @@
   },
   "title": "Buscador del estatus de su solicitud",
   "welcome": "Buscador del estatus de su solicitud",
-  "change-locale": "In English",
+  "errors": {
+    "not-found": {
+      "label": "We can't find that page.",
+      "entry": "The link you clicked may be broken or the page may have been removed."
+    },
+    "button": "Return to UI Home",
+    "server-error": {
+      "label": "There’s an error on our end.",
+      "entry": "We are unable to perform your request."
+    }
+  },
```

Comment:
As discussed in the slack, we're not translating these yet

Reply:
@nicoleslaw Do you want a ticket reminder for this?

Reply:
Nicole already created one! https://github.com/cagov/ui-claim-tracker/issues/452